bluesky/callbacks/fitting.py

```diff
@@ -282,7 +282,9 @@ def compute(self):
             self.cen = np.mean(_cen_list)
 
         if len(_cen_list) == 2:
-            self.fwhm = float(_cen_list[1] - _cen_list[0])
+            self.fwhm = {'fwhm': float(_cen_list[1] - _cen_list[0]),
```

Comment:
@mrakitin If you refactor this PR onto #888, can you use `self.fwhm` and `self.crossings` instead?

Reply:
Oh, cool! I didn't realized that looking at the current master. Should I wait until #888 is merged?

Reply:
That's probably easiest.

Reply:
Is this going to break existing code that expects self.fwhm to be a number, not a dict?

Reply:
@prjemian It would, but I don't believe it will be necessary to make this change in the end.